Lint errors

As seen on https://github.com/projectdiscovery/nuclei/actions/runs/18927835215/job/54038600627

```
  Running [/home/runner/golangci-lint-2.6.0-linux-amd64/golangci-lint config path] in [/home/runner/work/nuclei/nuclei] ...
  Running [/home/runner/golangci-lint-2.6.0-linux-amd64/golangci-lint run  --timeout=30m] in [/home/runner/work/nuclei/nuclei] ...
  Error: lib/example_test.go:2:1: buildtag: +build line is no longer needed (govet)
  // +build !race
  ^
  Error: pkg/operators/common/dsl/dsl.go:118:50: printf: missing ... in args forwarded to print-like function (govet)
  		gologger.Debug().Msgf("print_debug value: %s", fmt.Sprint(args))
  		                                               ^
  Error: pkg/utils/json/json.go:2:1: buildtag: +build line is no longer needed (govet)
  // +build linux darwin windows
  ^
  Error: pkg/utils/json/json.go:3:1: buildtag: +build line is no longer needed (govet)
  // +build amd64 arm64
  ^
  4 issues:
  * govet: 4
  
  Error: issues found
  Ran golangci-lint in 7490ms
```

--- a/lib/example_test.go
+++ b/lib/example_test.go
@@ -1,5 +1,4 @@
 //go:build !race
-// +build !race
 
 package nuclei_test
 

--- a/pkg/operators/common/dsl/dsl.go
+++ b/pkg/operators/common/dsl/dsl.go
@@ -115,7 +115,7 @@ func init() {
 	}))
 
 	dsl.PrintDebugCallback = func(args ...interface{}) error {
-		gologger.Debug().Msgf("print_debug value: %s", fmt.Sprint(args))
+		gologger.Debug().Msgf("print_debug value: %s", fmt.Sprint(args...))
 		return nil
 	}
 

--- a/pkg/scan/events/scan_noop.go
+++ b/pkg/scan/events/scan_noop.go
@@ -1,5 +1,4 @@
 //go:build !stats
-// +build !stats
 
 package events
 

--- a/pkg/scan/events/stats_build.go
+++ b/pkg/scan/events/stats_build.go
@@ -1,5 +1,4 @@
 //go:build stats
-// +build stats
 
 package events
 

--- a/pkg/utils/json/json.go
+++ b/pkg/utils/json/json.go
@@ -1,6 +1,4 @@
 //go:build (linux || darwin || windows) && (amd64 || arm64)
-// +build linux darwin windows
-// +build amd64 arm64
 
 package json
 

--- a/pkg/utils/json/json_fallback.go
+++ b/pkg/utils/json/json_fallback.go
@@ -1,5 +1,4 @@
 //go:build !(linux || darwin || windows) || !(amd64 || arm64)
-// +build !linux,!darwin,!windows !amd64,!arm64
 
 package json
 

